crates/bitwarden/src/client/encryption_settings.rs

```diff
@@ -119,7 +119,7 @@ impl EncryptionSettings {
     pub(crate) fn encrypt(&self, data: &[u8], org_id: &Option<Uuid>) -> Result<EncString> {
         let key = self.get_key(org_id).ok_or(CryptoError::NoKeyForOrg)?;
 
-        let dec = encrypt_aes256(data, key.mac_key, key.key)?;
+        let dec = encrypt_aes256_hmac(data, key.mac_key.unwrap(), key.key)?;
```

Comment:
Should we use something like `let mac_key = key.mac_key.ok_or(CryptoError::InvalidMac)?;` instead of `unwrap()`? Or can we be sure that the `mac_key` will always be `Some`?